libs/build-nss-desktop.sh

```diff
@@ -33,6 +33,13 @@ elif [[ -n "${CROSS_COMPILE_TARGET}" ]]; then
 elif [[ "$(uname -s)" == "Darwin" ]]; then
   DIST_DIR=$(abspath "desktop/darwin/nss")
   TARGET_OS="macos"
+  #We need this variable for switching libs based on different macos archs (M1 vs Intel)
```

Comment:
nit: here and below, please add a space between the comment char and the comment itself